src/UbuntuToolkit/sortfiltermodel.cpp

```diff
@@ -36,14 +36,13 @@ UT_NAMESPACE_BEGIN
  *
  * Example usage:
  * \qml
- * import QtQuick 2.4
- * import Ubuntu.Components 1.2
- * import Ubuntu.Components.ListItems 1.1
- *
+ * import QtQuick 2.9
+ * import Ubuntu.Components 1.3
+ * 
  * MainView {
  *     width: units.gu(80)
  *     height: units.gu(40)
- *
+ * 
```

Comment:
There are several lines in this file where the only addition was a trailing space, like this. Please remove these.